srcpkgs/pam_zfscrypt/INSTALL.msg

```diff
@@ -0,0 +1 @@
+Refer to https://github.com/BenKerry/zfscrypt#setup for configuration details.
```

Comment:
Don't use `INSTALL.msg` for things like that, they are already not being read and adding more doesn't improve the situation.

Reply:
Noted. Removed!